src/app/public/modules/dropdown/dropdown.component.spec.ts

```diff
@@ -743,6 +743,47 @@ describe('Dropdown component', () => {
         verifyTriggerButtonHasFocus(false);
       }));
     });
+  });
+
+  describe('focus properties', () => {
+    type focusProperty = 'buttonIsFocused' | 'menuIsFocused';
+
+    function validateFocus(hasFocus: boolean, focusedEl?: HTMLElement, focusPropertyName: focusProperty = 'buttonIsFocused'): void {
+      if (hasFocus && focusedEl) {
+        focusedEl.focus();
+      }
+
+      expect(component.dropdown[focusPropertyName]).toBe(hasFocus);
+    }
+
+    function validateMenuFocus(hasFocus: boolean, focusedEl?: HTMLElement): void {
+      openOrClosePopoverWithButtonClick();
+
+      validateFocus(hasFocus, focusedEl, 'menuIsFocused');
 
+      openOrClosePopoverWithButtonClick(true);
+    }
+
+    it('buttonIsFocused and menuIsFocused reflect the state of focus', fakeAsync(() => {
```

Comment:
The `expectation` parameter of `it()` should be rephrased so that when combined with the `description` parameter of the parent `describe()` functions, it makes a complete sentence stating the expectation, e.g. "should reflect the state of focus" (as in "Dropdown component focus properties should reflect the state of focus").